addons/account/views/account_move_views.xml

```diff
@@ -1075,7 +1075,7 @@
                                                domain="[('partner_id', '=', bank_partner_id)]"
                                                attrs="{'readonly': [('state', '!=', 'draft')]}"/>
                                         <field name="qr_code_method"
-                                               attrs="{'invisible': [('display_qr_code', '=', False)]}"/>
+                                               attrs="{'invisible': ['|', ('display_qr_code', '=', False), ('move_type', '=', 'out_refund')]}"/>
```

Comment:
This should be a `!=` or a `not in`, otherwise it will be displayed for purchase documents as well